src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -64,8 +64,10 @@
     </div>
   </td>
   <td class="cellProductLocation{% if not isColumnLocationDisplayed %} d-none{% endif %}">{{ product.location }}</td>
-  <td>
-    {{ product.quantityRefunded }}{% if product.amountRefunded %} ({{ product.amountRefunded }} {{ 'Refunded'|trans({}, 'Admin.Orderscustomers.Feature') }}){% endif %}
+  <td class="cellProductRefunded{% if not isColumnRefundedDisplayed %} d-none{% endif %}">
```

Comment:
Be careful, you need to add this in the product add controller as well, we had a bug related to this when `isColumnLocationDisplayed` was integrated Because when the product is added via ajax, the controller returns the displayed row and uses this template directly, so the variable will be missing as is

Reply:
Done